### PR TITLE
add signature configuration and deployment docs

### PR DESCRIPTION
**Pull request by aweiteka**
Signed-off-by: Aaron Weitekamp <aweiteka@redhat.com>

@adellape, please take a look at this work in progress. There are some opportunities for cross linking with your dev section and also for a section (in security>build?) for actually signing images. :)

### CODE REVIEW

**Comment by aweiteka** [comment]
Ready for review:

* https://people.redhat.com/aweiteka/docs/preview/20170720/security/deployment.html#security-deployment-from-where-images-deployed
* https://people.redhat.com/aweiteka/docs/preview/20170720/admin_guide/image_signatures.html
* https://people.redhat.com/aweiteka/docs/preview/20170720/install_config/install/host_preparation.html#enabling-image-signature-support

File: security/deployment.adoc [hunk 1]
```diff
@@ -48,6 +48,135 @@ $ oc set triggers dc/frontend \
...
+                    "keyType": "GPGKeys",
+                    "keyPath": "/etc/pki/example.com/pubkey"
+                }
+            ],
+            "172.30.1.1:5000": [{"type": "insecureAcceptAnything"}]
+        }
+    }
+}
+----
+====
```
Comment (adellape) [inline security/deployment.adoc]: Where does this file go? And does it need to be JSON or can it be YAML?

File: security/deployment.adoc [hunk 2]
```diff
@@ -48,6 +48,135 @@ $ oc set triggers dc/frontend \
...
+
+This policy enforces the following rules:
+
+.  Require images from the Red Hat registry are signed by the Red Hat public key
+.  Require images from the internal registry in the 'openshift' namespace are signed by the Red Hat public key
+.  Require images from the internal registry in the 'production' namespace are signed by example.com's public key
```
Comment (adellape) [inline security/deployment.adoc]: Trying to call the "internal registry" (or any other permutation) just the "OpenShift Container Registry".

File: security/deployment.adoc (hunk 1 above)
Comment (aweiteka) [inline security/deployment.adoc]: Updated.

**Comment by adellape** [comment]
Continuing via https://github.com/openshift/openshift-docs/pull/4937.

**Pull request closed without merging**